python_modules/automation/parse_spark_configs.py

```diff
@@ -53,10 +186,12 @@ def split_path(self):
         return self.path.split('.')
 
     def print(self, printer):
+        config_type = CONFIG_TYPES.get(self.path, CONFIG_TYPE.STRING).value
+
         printer.append('Field(')
         with printer.with_indent():
             printer.line('')
-            printer.line('String,')
+            printer.line('{},'.format(config_type))
```

Comment:
fwiw I'm paranoid about unnamed replacement fields in format strings

Reply:
good catch, fixed